src/app/zap-templates/templates/app/cluster-enums-check.zapt

```diff
@@ -0,0 +1,56 @@
+{{> header}}
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-enums.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+{{#zcl_clusters}}
+{{#zcl_enums}}
+static CHIP_ERROR __attribute__((unused)) CheckValidEnumValue({{asUpperCamelCase ../name}}::{{asType label}} val)
+{
+    switch (val) {
+{{#if (isWeaklyTypedEnum label)}}
+// Need to convert consumers to using the new enum classes, so we
+// don't just have casts all over.
+#ifdef CHIP_USE_ENUM_CLASS_FOR_IM_ENUM
+{{/if}}
+    using EnumType = {{asUpperCamelCase ../name}}::{{asType label}};
+    {{#zcl_enum_items}}
+    case EnumType::k{{asUpperCamelCase label}}:
+    {{/zcl_enum_items}}
+{{#if (isWeaklyTypedEnum label)}}
+#else // CHIP_USE_ENUM_CLASS_FOR_IM_ENUM
+    {{#zcl_enum_items}}
+    case EMBER_ZCL_{{asDelimitedMacro parent.label}}_{{asDelimitedMacro label}}:
+    {{/zcl_enum_items}}
+#endif // CHIP_USE_ENUM_CLASS_FOR_IM_ENUM
+{{/if}}
+        return CHIP_NO_ERROR;
+    default:
+        return CHIP_ERROR_UNKNOWN_ENUM_VALUE;
+    }
+}
+{{/zcl_enums}}
+
+{{#zcl_bitmaps}}
+static CHIP_ERROR __attribute__((unused)) CheckValidEnumValue({{asUpperCamelCase ../name}}::{{asType label}} val)
+{
+    switch (val) {
+    using EnumType = {{asUpperCamelCase ../name}}::{{asType label}};
```

Comment:
This is not an enum, right?

Reply:
per offline discussion, just removed bitmap part. thanks